flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/MeasurementInfoProviderTest.java

```diff
@@ -56,9 +78,26 @@ public void simpleTestGetMetricInfo() {
         assertEquals(
                 String.join("" + MeasurementInfoProvider.SCOPE_SEPARATOR, logicalScope, metricName),
                 info.getName());
-        assertThat(info.getTags(), hasEntry("A", "a"));
-        assertThat(info.getTags(), hasEntry("B", "b"));
-        assertThat(info.getTags(), hasEntry("C", "c"));
+        assertThat(info.getTags(), hasEntry(KEY1, VALUE1));
+        assertThat(info.getTags(), hasEntry(KEY2, VALUE2));
+        assertThat(info.getTags(), hasEntry(KEY3, VALUE3));
         assertEquals(3, info.getTags().size());
     }
+
+    private static Map<String, String> generateVariables(boolean isDelimited) {
+        Map<String, String> variables = new HashMap<>();
+        variables.put(String.format("<%s>", make(KEY1, isDelimited)), make(VALUE1, isDelimited));
+        variables.put(String.format("<%s>", make(KEY2, isDelimited)), make(VALUE2, isDelimited));
+        variables.put(String.format("<%s>", make(KEY3, isDelimited)), make(VALUE3, isDelimited));
+        return variables;
+    }
+
+    private static String make(String value, boolean isDelimited) {
+        return isDelimited ? delimit(value) : value;
```

Comment:
This is all overly complicated.
All we need is one test that does this:
```
variables = ("key\n", "value\n");
metricGroup = ...
info = provider.getMetricInfo(...)
assertThat(info.getTags(), containsEntry(key, value));
```

Reply:
ok